assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -156,6 +157,7 @@ const SnackbarMessage = ({
       {...props}
     >
       <Message
+        aria-label={ariaLabel}
```

Comment:
`aria-label` was being added to the container too, so the message was being read twice.